cmd/grafana-reporter/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/gorilla/mux"
 	"github.com/izakmarais/reporter"
```

Comment:
Change this to "github.com/IzakMarais/reporter" ( Capital I and M ) so that Travis check does not pull izakmarais:master from git and use it in place of this code.